README.md

```diff
@@ -14,6 +14,9 @@
 
 
 ## Overview
+
+:arrow_right:[The doc is now available here](https://owkin.github.io/FLamby):arrow_left:
```

Comment:
I'm not sure this is a sustainable way to present the documentation, in the sense that yes it's "now" available here but in 6 months it won't be news anymore. But feel free to do as you want.

Reply:
I agree with @mandreux-owkin . I also suggest adding the link to 'about' section of the repo (I would do it but I have no rights). It will be easier find